### PR TITLE
Vet clinic database: add "join table" for visits

### PR DESCRIPTION
**Pull request by mrprotocoll**
### The above updates were made

- Add specialization table
- Add visits table
- update animals table to include the primary key
- Add vets table
- Add records to vets, specialization and visits table
- Add queries to perform operations on tables with many-to-many relationship

### Screenshot of query results
image

image

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -40,3 +40,36 @@ INSERT INTO species (name) VALUES ('Digimon');
...
+INSERT INTO vets (name, age, date_of_graduation) 
+VALUES ('William Tatcher', 45, 'Apr 23, 2000'),
+('Maisy Smith', 26, 'Jan 17, 2019'),
+('Stephanie Mendez', 64, 'May 4, 1981'),
+('Jack Harkness', 38, 'Jun 8, 2008');
+
+-- insert data into the specializations table
+INSERT INTO specializations (vets, species) VALUES (1, 1),
+(3, 2),
+(4, 2);
```
Comment (codecaiine) [inline data.sql]: - [ ] Good job so far 👏  But I think, you need to Insert the following data for specialties Both  Vet Stephanie Mendez is specialized in Digimon and Pokemon, and I can see only one query. Please make sure to add the missing query 👍

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @mrprotocoll  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Great schema implementation ✔️ 
- Good joining table queries set up ✔️ 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (mrprotocoll) [reply data.sql]: Hello @codecaiine , Thanks. I will make the changes and request a new review

**Review by alexander16108 — APPROVED** [review]
Hi @mrprotocoll , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Great schema implementation ✔️
Good joining table queries set up ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**